test/e2e/beta/drizzle.spec.js

```diff
@@ -0,0 +1,286 @@
+const path = require('path')
+const assert = require('assert')
+const webdriver = require('selenium-webdriver')
+const { By, until } = webdriver
+const {
+  delay,
+  buildChromeWebDriver,
+  buildFirefoxWebdriver,
+  installWebExt,
+  getExtensionIdChrome,
+  getExtensionIdFirefox,
+} = require('../func')
+const {
+  checkBrowserForConsoleErrors,
+  closeAllWindowHandlesExcept,
+  findElement,
+  findElements,
+  loadExtension,
+  openNewPage,
+  verboseReportOnFailure,
+  waitUntilXWindowHandles,
+} = require('./helpers')
+
+describe('MetaMask', function () {
+  let extensionId
+  let driver
+
+  const tinyDelayMs = 200
+  const regularDelayMs = tinyDelayMs * 2
+  const largeDelayMs = regularDelayMs * 2
+
+  this.timeout(0)
+  this.bail(true)
+
+  before(async function () {
+    switch (process.env.SELENIUM_BROWSER) {
```

Comment:
This selenium bootstrapping should be extracted into a utility (I'm sure it already is somewhere). You shouldn't need more than a line or two for this generic setup

Reply:
Sounds good! l’ll handle tomorrow.